adminSiteClient/GrapherConfigGridEditor.tsx

```diff
@@ -91,6 +94,61 @@ import {
 import { Query, Utils as QbUtils } from "react-awesome-query-builder"
 // types
 import { SimpleField, Config, ImmutableTree } from "react-awesome-query-builder"
+import codemirror from "codemirror"
+import { UnControlled as CodeMirror } from "react-codemirror2"
+import jsonpointer from "json8-pointer"
+import { EditorColorScaleSection } from "./EditorColorScaleSection.js"
+import { MapChart } from "../grapher/mapCharts/MapChart.js"
+
+function HotColorScaleRenderer(props: Record<string, unknown>) {
+    return <div>Color scale</div>
```

Comment:
One last thing: you could use a different color (e.g. grey) here to differentiate this cell from "normal" string cells.